packages/react-native-usb/README.md

```diff
@@ -0,0 +1,27 @@
+# react-native-usb
+
+React Native WebUSB implementation
+
+# Installation in managed Expo projects
+
+For [managed](https://docs.expo.dev/archive/managed-vs-bare/) Expo projects, please follow the installation instructions in the [API documentation for the latest stable release](#api-documentation). If you follow the link and there is no documentation available then this library is not yet usable within managed projects &mdash; it is likely to be included in an upcoming Expo SDK release.
+
+# Installation in bare React Native projects
+
+For bare React Native projects, you must ensure that you have [installed and configured the `expo` package](https://docs.expo.dev/bare/installing-expo-modules/) before continuing.
+
+### Add the package to your npm dependencies
+
+```
+npm install react-native-usb
+```
+
+### Configure for iOS
+
+Run `npx pod-install` after installing the npm package.
```

Comment:
I would add here, that it is not implemented.

Reply:
done